### ISSUE TITLE
Can't Accept Friend Requests

### ISSUE THREAD

**Issue by Pig0Brine** [Sopur/Discord-user-bots#38]
When trying to accept friend requests, I get this error:
`{ code: 50109, message: 'The request body contains invalid JSON.' }`

**Comment by Sopur** [comment]
What is the code you're using to accept friend requests?

**Comment by Pig0Brine** [comment]
This: 


```
// Auto Accept Friend Requests
client.on.relationship_add = async function(relationship) {
    // console.log(relationship);

    if (relationship.type === 3) {
        await delay(5000);

        console.log(`Added ${relationship.user.username}#${relationship.user.discriminator} as a friend!`);
        console.log(await client.accept_friend_request(relationship.id));
    }
}
```

**Comment by Jaydenha09** [comment]
Use another npm module called discord.js-selfbot-v13 which is much better
They have complete documents and updates very often
[GitHub](https://github.com/aiko-chan-ai/discord.js-selfbot-v13)
[Document Website](https://discordjs-self-v13.netlify.app/)

**Comment by Sopur** [comment]
This problem was fixed in DUB version 2.0.0.
